packages/api-server/src/db/helpers.ts

```diff
@@ -223,75 +221,21 @@ export function filterLogsByAddress(
   return result;
 }
 
-export enum QueryRoundStatus {
-  keepGoing,
-  stop,
-}
-
-export interface ExecuteOneQueryResult {
-  status: QueryRoundStatus;
-  data: any[];
-}
-
 export function getDatabaseRateLimitingConfiguration() {
   const MAX_QUERY_NUMBER = envConfig["maxQueryNumber"]
     ? +envConfig["maxQueryNumber"]
     : DEFAULT_MAX_QUERY_NUMBER;
+  // TODO set query timeout
   const MAX_QUERY_TIME_MILSECS = envConfig["maxQueryTimeInMilliseconds"]
     ? +envConfig["maxQueryTimeInMilliseconds"]
     : DEFAULT_MAX_QUERY_TIME_MILSECS;
-  const MAX_QUERY_ROUNDS = envConfig["maxQueryRounds"]
```

Comment:
since maxQueryRounds is removed, please update `env-config.ts` and `README.md` as well